apps/emqx_gateway/test/emqx_gateway_api_SUITE.erl

```diff
@@ -561,7 +550,40 @@ t_authn_fuzzy_search(_) ->
 
     {204, _} = request(delete, "/gateways/stomp/authentication"),
     {204, _} = request(get, "/gateways/stomp/authentication"),
-    {204, _} = request(delete, "/gateways/stomp").
+    ok.
+
+%%--------------------------------------------------------------------
+%% Helpers
+
+await(What, How, Timeout) when Timeout > 0 ->
```

Comment:
We also have [`?retry/3`](https://kafka4beam.github.io/snabbkaffe-docs/1.0.0-1-ga1bcd97/snabbkaffe.html#retry/3) from Snabbkaffe.

Reply:
Hm, it's not quite the same though. First, I'd still have to manually make my calls throw an exception (i.e. wrap them), second it's retries vs timeouts. But then maybe retries makes more sense anyway. 🤔

Reply:
True, the call would have to contain some assertion that would fail to be retried.